relay/kusama/src/lib.rs

```diff
@@ -1803,6 +1831,13 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
 	}
 }
 
+parameter_types! {
+	// This is used to limit max pools that migrates in the runtime upgrade.  This is set to
+	// existing_pool_count plus ~15 to also account for any new pools getting created before the
+	// migration is actually executed.
+	pub const MaxPoolsToMigrate: u32 = 200;
```

Comment:
Same here

Reply:
After https://github.com/pandres95/runtimes/pull/5, this part was removed.

Closing conversation for now.